### ISSUE TITLE
Outline client stopped working since yesterday on Windows 10!

### ISSUE THREAD

**Issue by mohammad4x** [Jigsaw-Code/outline-client#1065]
Outline client stopped working since yesterday on Windows 10! Outline manager works fine but Outline client has a connection error when i click on "Connect" button. The error mentioned is below :
![Capture](https://user-images.githubusercontent.com/16001221/128507075-12d74f44-1ed5-4484-9367-a050e52be492.PNG)

**Comment by mohammad4x** [comment]
I uninstalled the app and downloaded the latest client version from https://getoutline.org/ and installed it, nothing changed :(

**Comment by kutato69** [comment]
Win 11 same too

**Comment by kutato69** [comment]
> I uninstalled the app and downloaded the latest client version from https://getoutline.org/ and installed it, nothing changed :(

I fixed by install  C++ library https://www.microsoft.com/en-gb/download/details.aspx?id=48145

**Comment by mohammad4x** [comment]
tried the C++ library, already had a newer version so it did not install it for me. still now working :(

**Comment by joeytwiddle** [comment]
Possible related to: https://github.com/Jigsaw-Code/outline-client/issues/782#issuecomment-630983790

Do you think this has broken because Windows Update has upgraded the library?

**Comment by evil-shrike** [comment]
I'm glad it's solved for you, but I'm having exact same issue. Outline Client has stopped working in Win10. It works fine in the same network on Android and MacOS. I've installed VC++ redist x86/x64 already, and reinstalled the client a couple of times. It doesn't help.

**Comment by mohammad4x** [comment]
The Client has problems with virtual network adapters when installed alongside other vpn applications. everytime I installed OpenVpn, network adapters get buggy on both (Openvpn & Outline) apps. On windows 11, this problem shows itself more! On windows 10 sometimes it works (randomly). Since I no longer need OpenVpn, I uninstalled them both and installed Outline client and have no issues what so ever.
